Docs: Include instructions for minikube (#122)
kinvolk/inspektor-gadget: Documentation/install.md

- `flatcar_edge`: Use a custom `runc` version shipped with Flatcar Container Linux Edge.
- `podinformer`: Use a K8s controller to get information about new pods. This option is racy and the first events produced by a container could be lost. This mode is selected when `auto` is used and the above modes are not available.
- `ldpreload`: Adds an entry in `/etc/ld.so.preload` to call a custom shared library that looks for `runc` calls and dynamically adds the needed OCI hooks to the cointainer `config.json` specification. Since this feature is highly experimental, it'll not be considered when `auto` is used.

### Specific Information for Different Platforms

This section explains the additional steps that are required to run Inspektor Gadget in some platforms.

#### Minikube

> To deploy Inspektor Gadget in Minikube it's needed to install the kernel headers manually before this [issue](https://github.com/kubernetes/minikube/issues/8556) is solved.

Reviewer: ```suggestion
To deploy Inspektor Gadget in Minikube the kernel headers need to be installed manually (until this [issue](https://github.com/kubernetes/minikube/issues/8556) is solved).
```

```
# Docker driver is not supported, use a VM driver like kvm2
minikube config set driver kvm2

# Set this memory to be able to extract kernel headers without errors
minikube config set memory 4096

> # Use a special minikube iso

Reviewer: special how? Can you clarify what's special about this iso?
